### DISCUSSION TITLE
Erreur survient avant de pouvoir saisir les informations CB en production

### DISCUSSION THREAD

**Discussion by fkoca** [PayGreen/documentation#42] [Q&A]
Bonjour, plusieurs utilisateurs finaux nous remontent l'erreur suivante qui survient aléatoirement: Quand il choisisse le moyen de paiement, les champs de saisies des informations sont masqués et il est affiché "Une erreur est survenue". Pourtant quand je test de mon coté, je n'arrive pas à reproduire l'erreur. Pourriez-vous m'aider à résoudre ce problème svp? ID boutique: sh_96eeb68b4b95481d9bf4507fec676dc9

**Comment by gaelDolle** [comment]
Bonjour,

Merci de créer un ticket sur notre service client afin d'analyser votre soucis si celui-ci persiste.

[Service Client PayGreen](https://paygreen.atlassian.net/servicedesk/customer/portal/9)

Cordialement
